csharp/src/Drivers/Databricks/Telemetry/Util.cs

```diff
@@ -0,0 +1,101 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using Apache.Arrow.Adbc.Drivers.Databricks.Telemetry.Enums;
+
+namespace Apache.Arrow.Adbc.Drivers.Databricks.Telemetry
+{
+    internal class Util
+    {
+        private static string DRIVER_VERSION = "1.0.0";
```

Comment:
These should follow the same pattern that the other drivers in the repo do. The name is the DLL name and the version is the assembly version. See https://github.com/apache/arrow-adbc/blob/15e161e3657563f0c49decd0b3eefa30ed6673e8/csharp/src/Drivers/BigQuery/BigQueryUtils.cs#L38.